build.sbt

```diff
@@ -218,11 +229,11 @@ lazy val `interpreter-play`: sbtcrossproject.CrossProject =
     .crossType(CrossType.Full)
     .settings(commonSettings)
     .configure(_.dependsOn(core.jvm, `common-web`.jvm))
-    .configurePlatform(Play25) {_.settings(
-      name := "interpreter-play25",
-      scalaVersion := allCrossScala.find(_.startsWith("2.11")).get,
-      crossScalaVersions := allCrossScala.filter{_.startsWith("2.11")}
-    )}
+    // .configurePlatform(Play25) {_.settings(
+    //   name := "interpreter-play25",
+    //   scalaVersion := allCrossScala.find(_.startsWith("2.11")).get,
+    //   crossScalaVersions := allCrossScala.filter{_.startsWith("2.11")}
+    // )}
```

Comment:
There's quite a bit of commented out stuff in build.sbt, do you need all of it?
Some of these versions are already quite old e.g. cats-core, do you want to run with this for now?